apps/web/src/pages/Teams/IndexTeamPage/components/TeamTable/TeamTable.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import fakeTeams from "@common/fakeData/fakeTeams";
+
+import { TeamTable } from "./TeamTable";
+
+const mockTeams = fakeTeams();
+
+export default {
+  component: TeamTable,
+  title: "Tables/Department Table",
+} as ComponentMeta<typeof TeamTable>;
```

Comment:
think you'll need a different title for the story here
same as this one

https://github.com/GCTC-NTGC/gc-digital-talent/blob/d9a45927cb53323313c5366b8feb200421919cb9/apps/web/src/pages/Departments/components/DepartmentTable.stories.tsx#L10-L13

it's making weirdness in the chromatic diff

![image](https://user-images.githubusercontent.com/40485260/216477897-39cf0541-6a84-4b93-861c-2414dc80996a.png)

Reply:
heh, missed copy + past 😅 

Fixed in [fix copy paste typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5555/commits/49fec9aad78ce61760362811de2ff0b8a168ef67)